### PR TITLE
implements PartedModule in Symfony2 module with services part

### PR DESCRIPTION
**Pull request by laszlo-karpati**
Codeception/Codeception#2435

Codeception/Codeception#2470

Usage example:

```yaml
class_name: AcceptanceTester
modules:
    enabled:
        - Symfony2:
            part: SERVICES
            app_path: '../../../app'
            var_path: '../../../app'
        - Doctrine2:
            depends: Symfony2
            cleanup: false
        - Asserts:
        - WebDriver:
            url: http://your-url.com
            host: '127.0.0.1'
            browser: phantomjs
            window_size: 1280x800
            request_timeout: 30
            debug_log_entries: 100
            wait: 1
            capabilities:
                webStorageEnabled: true
                handlesAlerts: true
        - \FrontendAdminBundle\Helper\Acceptance
```

### CODE REVIEW

File: src/Codeception/Module/Symfony2.php [hunk 1]
```diff
@@ -86,6 +87,14 @@ class Symfony2 extends Framework implements DoctrineProvider
     ];
 
     /**
+     * @return array
+     */
+    public function _parts()
+    {
+        return ['serives'];
```
Comment (raistlin) [inline src/Codeception/Module/Symfony2.php]: Shouldn't this be 'services' ?

**Comment by laszlo-karpati** [comment]
@raistlin : 
Sure, thanks! Fixed.

**Comment by DavertMik** [comment]
Good one. Thank you

**Comment by DavertMik** [comment]
@janhenkgerritsen should this be ported to Laravel as well? 

Also this should be documented somehow.

**Comment by janhenkgerritsen** [comment]
@DavertMik the Laravel modules already implement the `PartedModule` interface.

**Pull request merged**